src/common/utility.ts

```diff
@@ -448,3 +448,7 @@ export const handleRedirect = (residence, current_client_country) => {
 export const isLocalhost = () => !!(isBrowser() && process.env.NODE_ENV === 'development')
 
 export const isTestlink = () => !!(isBrowser() && window.location.hostname.includes('binary.sx'))
+
+export const getDomainName = (hostname) => {
+    return hostname.substring(hostname.lastIndexOf('.', hostname.lastIndexOf('.') - 1) + 1)
```

Comment:
Let's not passed the `hostname` as parameter, it will always be thesame anyways. fetch the hostname inside `getDomainName` function

Reply:
okay